iron-input.html

```diff
@@ -238,11 +244,12 @@
         return;
 
       // Check the pattern either here or in `_onInput`, but not in both.
-      this._patternAlreadyChecked = true;
+      this._patternAlreadyChecked = false;//true;
 
       var thisChar = String.fromCharCode(event.charCode);
       if (this._isPrintable(event) && !regexp.test(thisChar)) {
         event.preventDefault();
+        this._announceInvalidCharacter('Invalid character ' + thisChar + ' not entered.');
```

Comment:
I talked to @lpalmaro, and that's the announcement we both agreed one (so that it's less verbose). Also you want to imply that you did a thing, and it was invalid, hence the "entered". You entered it, it was bad. WDYT?